src/components/select/simple-select/simple-select-sizes.stories.mdx

```diff
@@ -10,7 +10,7 @@ import { Select, Option } from '../';
 <Preview>
   <Story name="small" parameters={{ info: { disable: true }}} >
     <div style={{ height: '250px' }}>
-      <Select name='small' id='small' defaultValue='3' size='small'>
+      <Select name='small' id='small' defaultValue='3' size='small' isRequired>
```

Comment:
Please can you add a dedicated story for `required`. The prop should be `required` or `aria-required` not `isRequired`.

Reply:
Done